.gitignore

```
@@ -0,0 +1,5 @@
+# CMake artifacts
+build
```

The following pattern handles more cases, including ones specific to some IDEs:
```suggestion
/*build*
```